test/unit/transformers/protocol-token-wrapper-transformer.spec.ts

```diff
@@ -189,6 +189,83 @@ describe('ProtocolTokenWrapperTransformer', () => {
     });
   });
 
+  describe('transformToExpectedUnderlying', () => {
+    when('function is called', () => {
+      given(async () => {
+        // We are setting balance to the transformer, to simulate a withdraw from the wToken
+        setBalance(transformer.address, AMOUNT_TO_MAP);
+        await transformer.transformToExpectedUnderlying(wToken.address, [{ underlying: PROTOCOL_TOKEN, amount: AMOUNT_TO_MAP }], RECIPIENT);
+      });
+      then('wToken is taken from caller', () => {
+        expect(wToken.transferFrom).to.have.been.calledOnceWith(signer.address, transformer.address, AMOUNT_TO_MAP);
+      });
+      then('wToken is called correctly', () => {
+        expect(wToken.withdraw).to.have.been.calledOnceWith(AMOUNT_TO_MAP);
+      });
+      then('protocol token is transferred correctly to the recipient', async () => {
+        const balance = await ethers.provider.getBalance(RECIPIENT);
+        expect(balance).to.equal(AMOUNT_TO_MAP);
+      });
+      then('protocol token is removed from the transformer', async () => {
+        const balance = await ethers.provider.getBalance(transformer.address);
+        expect(balance).to.equal(0);
+      });
+      then('spent dependent is returned correctly', async () => {
+        // We are setting balance to the transformer, to simulate a withdraw from the wToken
+        setBalance(transformer.address, AMOUNT_TO_MAP);
+        const spentDependent = await transformer.callStatic.transformToExpectedUnderlying(
+          wToken.address,
+          [{ underlying: PROTOCOL_TOKEN, amount: AMOUNT_TO_MAP }],
+          RECIPIENT
+        );
+        expect(spentDependent).to.equal(AMOUNT_TO_MAP);
+      });
```

Comment:
```suggestion
      then('returns spent dependent value correctly', async () => {
        // We are setting balance to the transformer, to simulate a withdraw from the wToken
        setBalance(transformer.address, AMOUNT_TO_MAP);
        const spentDependent = await transformer.callStatic.transformToExpectedUnderlying(
          wToken.address,
          [{ underlying: PROTOCOL_TOKEN, amount: AMOUNT_TO_MAP }],
          RECIPIENT
        );
        expect(spentDependent).to.equal(AMOUNT_TO_MAP);
      });
```
At least to me it was kinda confusing, because `is returned` I started to think about change of balances, and not actual `return values;`